src/include/parser/expression/abstract_expression.h

```diff
@@ -66,6 +66,12 @@ class AliasType {
    */
   const std::string &GetName() const { return name_; }
 
+  /**
+   * Set the name of this alias
+   * @param name name to set alias to
+   */
+  void SetName(const std::string &name) { name_ = std::string(name); }
```

Comment:
`name_` is a `std::string` so there is no need to manually call the `std::string` constructor here.

Reply:
Changed